src/main/kotlin/io/zerobase/smarttracing/resources/Amazon.kt

```diff
@@ -0,0 +1,179 @@
+package io.zerobase.smarttracing.resources
+
+import io.zerobase.smarttracing.models.*
+import software.amazon.awssdk.core.exception.SdkException
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.ses.SesClient
+import javax.mail.Message
+import javax.mail.MessagingException
+import javax.mail.Session
+import javax.mail.internet.AddressException
+import javax.mail.internet.InternetAddress
+import javax.mail.internet.MimeMessage
+import javax.mail.internet.MimeMultipart
+import javax.mail.internet.MimeBodyPart
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+import java.nio.ByteBuffer
+import java.util.Properties
+import software.amazon.awssdk.core.SdkBytes
+import software.amazon.awssdk.services.ses.model.SendRawEmailRequest
+import software.amazon.awssdk.services.ses.model.RawMessage
+import io.zerobase.smarttracing.AmazonConfig
+
+// Currently only implements the AmazonSES
+class Amazon(private val config: AmazonConfig) {
+    private val session: Session
+    private val client: SesClient
+
+    init {
+        session = Session.getDefaultInstance(Properties())
+        client = SesClient.builder().region(Region.US_EAST_1).build()
```

Comment:
still no. this needs to be passed in. read up on the dependency inversion principle